spec/support/shoulda_matchers.rb

```diff
@@ -0,0 +1,6 @@
+Shoulda::Matchers.configure do |config|
```

Comment:
inside spec_helper

Reply:
then I should comment the following line `Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }` ?

Reply:
Nice, my bad i didn’t see the line